server/base/src/main/java/org/apache/accumulo/server/manager/state/TabletManagementScanner.java

```diff
@@ -70,7 +70,7 @@ public class TabletManagementScanner implements ClosableIterator<TabletManagemen
         final ClientTabletCache locator = ClientTabletCache.getInstance(context, tid);
         final Set<String> locations = new HashSet<>();
         final List<Range> failures = locator.findTablets(context, ALL_TABLETS_RANGE, (ct, r) -> {
-          locations.add(ct.getTserverLocation().orElseThrow());
+          locations.add(ct.getTserverLocation().orElse("UNASSIGNED"));
```

Comment:
Not that this fix is incorrect, but would it cause 2 extra threads in this case. One for the "UNASSIGNED" location and one if the failures list is not empty?  I wonder if it should be the following instead:

```suggestion
          if (ct.getTserverLocation().isPresent()) {
            locations.add(ct.getTserverLocation().orElseThrow());
          }
```